content/en/docs/setup/install/multicluster/primary-remote_multi-network/index.md

```diff
@@ -0,0 +1,351 @@
+---
+title: Install Primary-Remote on Different Networks
+description: Install an Istio mesh across primary and remote clusters on different networks.
+weight: 30
+icon: setup
+aliases:
+    - /docs/setup/kubernetes/multicluster-install/
+    - /docs/setup/kubernetes/multicluster/
+    - /docs/setup/kubernetes/install/multicluster/
+keywords: [kubernetes,multicluster]
+test: yes
+owner: istio/wg-environments-maintainers
+---
+Follow this guide to install the Istio control plane on `cluster1` (the
+{{< gloss >}}primary cluster{{< /gloss >}}) and configure `cluster2` (the
+{{< gloss >}}remote cluster{{< /gloss >}}) to use the control plane in
+`cluster1`. Cluster `cluster1` is on the `network1` network, while `cluster2`
+is on the `network2` network. This means there is no direct connectivity
+between pods across cluster boundaries.
+
+Before proceeding, be sure to complete the steps under
+[Before you begin](/docs/setup/install/multicluster#before-you-begin).
+
+In this configuration, cluster `cluster1` will observe the API Servers in
+both clusters for endpoints. In this way, the control plane will be able to
+provide service discovery for workloads in both clusters.
+
+Service workloads across cluster boundaries communicate indirectly, via
+dedicated gateways for [east-west](https://en.wikipedia.org/wiki/East-west_traffic)
+traffic. The gateway in each cluster must be reachable from the other cluster.
+
+Services in `cluster2` will reach the control plane in `cluster1` via the
+same east-west gateway.
+
+{{< image width="75%"
+    link="arch.svg"
+    caption="Primary and remote clusters on separate networks"
```

Comment:
Minor nit on the picture: It may not be clear to readers why Istiod can directly access the API Server in cluster2 but the services need to go through a gateway. Maybe we can show that as going through an ingress gateway or something?

Reply:
I discuss that in the requirements section in the top-level page. Maybe we need to reference it?

Reply:
Maybe it would be better if you drew the API Server boxes straddling the line around the mesh, like the Gateway boxes are done. That, combined with the explanation in the requirements section might be clear enough?

Reply:
done.

Reply:
I think you missed one, the one on the right side of the primary-remote diagram

Reply:
@frankbu I've updated the remaining diagrams. PTAL